src/Microsoft.AspNet.Mvc.Localization/IHtmlLocalizerOfT.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Mvc.Localization
+{
+    /// <summary>
+    /// An <see cref="IHtmlLocalizer"/> that provides strings for <see cref="TResourceSource"/>.
+    /// </summary>
+    /// <typeparam name="TResourceSource">The <see cref="System.Type"/> to provide strings for.</typeparam>
```

Comment:
- Isn't `TResourceSource` where the strings come _from_? Think there's a few other comments with similar confusions.
- Separately `TResource` seems specific enough for this type parameter.